lib/reqwestRequest.js

```diff
@@ -10,4 +10,20 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-define(['q', 'reqwest'], function(Q, reqwest) {
\ No newline at end of file
+var reqwest = require('reqwest');
+
+function reqwestRequest(method, url, args) {
+  var r = reqwest({
+    url: url,
+    method: method,
+    headers: args.headers,
+    data: JSON.stringify(args.data),
+    withCredentials: true
+  })
+  .then(function() {
```

Comment:
It doesn't. We need an xhr, but reqwest only returns the response body. This is the only way I've found to get the raw xhr.